doc/source/index.rst

```diff
@@ -8,14 +8,33 @@ Pyorbital
 
 Pyorbital is a python package to compute orbital parameters for satellites from
 TLE files as well as astronomical parameters of interest for satellite remote sensing.
-Currently pyorbital only supports low earth orbit satellites.
+Currently Pyorbital only supports low earth orbit satellites.
+
 
 Installation
 ------------
-Pyorbital comes with a file platforms.txt that maps satellite name to NORAD identifier.
-This file needs to be copied to the appropriate satpy etc directory ($PPP_CONFIG_DIR).
-It is wise to check it contains your satellites of interest. The NORAD identifier can
-be found as the first number of each line in the Two-Line Elements (eg. from celestrak).
+Pyorbital comes with a file *platforms.txt* that maps a satellite name to the NORAD identifier.
+
+This file already contain many low earth orbiting environmental or
+meteorological satellites and thus likely be sufficient for your purpose.
+
+But should it not contain your satellites of interest make a copy of the
+*platforms.txt* file and add the missing satellites and their NORAD identifiers and place
+the file in the directory pointed to by :envvar:`PYORBITAL_CONFIG_PATH`.
+
+The NORAD identifier can be found as the first number of each line in the
+Two-Line Elements files (eg. from `celestrak`_).
+
+Pyorbital comes with a small script ``check_platform.py`` to check whether a
```

Comment:
If this script is not part of the package then can this include a link to the script on github. If it is not part of the package then this sentence should be changed as "Pyorbital comes with" is not true.

If it does come with the package then `python -m pyorbital.check_platform ...` should be used instead of calling the script directly as that is how someone who has installed the package would access the script.

Reply:
This is done now